test/Utilities/model.jl

```diff
@@ -25,8 +25,28 @@ module TestExternalModel
         (NewFunction,),
         (),
         (),
-        ()
+        (),
     )
+    MathOptInterface.Utilities.@model(ExternalOptimizer,
+        (MathOptInterface.ZeroOne, NewSet,),
+        (),
+        (),
+        (),
+        (NewFunction,),
+        (),
+        (),
+        (),
+        true
+    )
+end
+
+@testset "Super-types" begin
+    model = TestExternalModel.ExternalModel{Float64}()
+    optimizer = TestExternalModel.ExternalOptimizer{Float64}()
+    @test isa(model, MOIU.AbstractModelLike{Float64})
+    @test !isa(model, MOIU.AbstractOptimizer{Float64})
+    @test !isa(optimizer, MOIU.AbstractModelLike{Float64})
+    @test isa(optimizer, MOIU.AbstractOptimizer{Float64})
```

Comment:
Also check with `MOI.AbstractOptimizer` to make sure the subtyping of the MOIU abstract types is correctly set up